### DISCUSSION TITLE
Problem in the problem description ?

### DISCUSSION THREAD

**Discussion by g-poveda** [esa/SpOC3#4] [Challenge 1: Torso Decompositions]
Hello, 
in the description https://optimize.esa.int/challenge/spoc-3-torso-decompositions/About
I wonder if this sentence is correct.

> We let the ordering enforces new edges by the following procedure: Scan the permutation from left to right and, whenever you encounter a node 
u with two outgoing edges to, say, v and w such that v and w are not yet connected by an edge, insert the edge 
**{u,v}** and orient it according to the permutation. 

Should it be **{v,w}** instead of **{u,v}** here ? because apriori (u,v) already exist in the graph, and also with the correction i propose i think i reproduce the red edges shown as example.
Thanks

**Comment by maxbannach** [comment]
Yes indeed, the new edge is {v,w}. We updated the text accordingly. 
Thanks for spotting it! :)
